src/main.ts

```diff
@@ -169,3 +174,67 @@ app.on('activate', () => {
     createWindow()
   }
 })
+
+ipcMain.on('app-ready', () => {
+  if (app.isPackaged && checkSignatureExists()) {
+    const feedURL = `${pkg.hostUrl}/${pkg.repoName}/${process.platform}-${
+      process.arch
+    }/${app.getVersion()}`
+
+    autoUpdater.setFeedURL({
+      url: feedURL,
+      serverType: 'default',
+    })
+    autoUpdater.checkForUpdates()
+    setInterval(() => {
+      autoUpdater.checkForUpdates()
+    }, 4 * 60 * 60 * 1000)
```

Comment:
nit: can we create a constant with the interval name, like 
```ts
const oneHour = 60 * 60
```

Reply:
Updated